server/swagger/swagger.yml

```diff
@@ -292,7 +335,9 @@ info:
   license:
     name: Custom
     url: https://github.com/mattermost/focalboard/blob/main/LICENSE.txt
-  title: Focalboard Server
+  title: |-
+    line /home/jespino/Projects/mattermost/focalboard/server/main/main.go:1:1
```

Comment:
:eyes:

Reply:
Yes, something weird there. I'll check it